packages/kolibri-common/strings/kolibriOnboardingGuideStrings.js

```diff
@@ -0,0 +1,63 @@
+import { createTranslator } from 'kolibri.utils.i18n';
+
+export const kolibriOnboardingGuideStrings = createTranslator('kolibriOnboardingGuideStrings', {
```

Comment:
I guess my only question here, is that all these strings seem to be specific to the learn experience, so why do we need to have them in `kolibri-common` and not in the learn app specifically?

Reply:
Good point. This is apart of the new method we’re trying out recommended to me by Marcella, putting all of the strings into one file per feature. Similar to how the `enhancedQuizManagementStrings` are specific to the coach plugin, I followed that setup. However, I don’t think this file necessarily has to be within the `kolibri-common` folder, it is just a convenient place to easily locate all of the new strings. @marcellamaki can probably correct me on this or add more context.

Reply:
I've thought a few times about moving the `enhancedQuizManagementStrings` into Coach. I'm keen on the idea sort of treating features as modules.

For example, in my case, "QuizCreation" could be it's own directory with it's relevant feature-specific state-management modules, strings files, constants, etc. I'm not super confident I'm fully considering possible downsides to this though.

Reply:
Yeah that's a good question, @rtibbles and I don't know actually if we made a concrete decision about where these feature files belong. Looking back through slack it seems like we only agreed that it would be one file per feature and namespaced, and I suggested to Liana that she follow the EQM model and did not give it much additional thought.

It does seem like for features that only impact one plugin, that the strings live in that plugin, but still in a strings folder with the feature namespace. And then, maybe when we have features that are cross-plugin, we can put those into `kolibri-common`.

Reply:
maybe the strings that are only used in one plugin could live in `[plugin]/src/strings/namespacedFeatureStringFile.js`

(with /strings/ being a new folder that we would need to create. 

What do you think @rtibbles @nucleogenesis @LianaHarris360 ?

Reply:
I hadn't realised that plugin specific strings were being put into kolibri-common at all, so yes, I think putting strings that are only used in one plugin in that plugin makes a lot more sense (both in this case and for the enhanced quiz management strings).

It was never the intention of kolibri-common to become a place where all new things get put, only for things that are explicitly used across multiple plugins.

Reply:
For EQM I had thought there were new strings that were used in both learn and coach, but actually there might not be any in learn, or if there are a few, I guess it's likely that they are not actually shared (i.e. the same string used in both places). So this would be good to check. @nucleogenesis let's open an issue for this and get it sorted before we get to string freeze?

@LianaHarris360 - so the next step would just be moving this file into Learn, and then I think it should be good to merge.